themes/theme-gmd/pages/Product/components/Media/index.jsx

```diff
@@ -12,24 +14,29 @@ import { ProductContext } from '../../context';
 const Media = ({ 'aria-hidden': ariaHidden }) => (
   <ProductContext.Consumer>
     {({ productId, variantId, characteristics }) => (
-      <Fragment>
-        {/* MediaSlider feature is currently in BETA testing.
-            It should only be used for approved BETA Client Projects */}
-        {isBeta() ? (
-          <ProductMediaSlider
-            productId={productId}
-            variantId={variantId}
-            characteristics={characteristics}
-            aria-hidden={ariaHidden}
-          />
-        ) : (
-          <ProductImageSlider
-            productId={productId}
-            variantId={variantId}
-            aria-hidden={ariaHidden}
-          />
-        )}
-      </Fragment>
+      <SurroundPortals
+        portalName={PORTAL_PRODUCT_MEDIA_SECTION}
+        portalProps={{ productId, variantId }}
+      >
+        <Fragment>
```

Comment:
The `Fragment` tag is redundant now, because there is the `SurroundPortals` tag in place now.
`Fragment` is a "dummy" component, which serves a non-functional purpose. It's purpose is to maintain JSX code syntax, because every component can and must return one single tag (with or without children), an array of elements or null, but `{isBeta() ? ( ... )}` is an expression, not a tag.

Reply:
I have removed the unneeded Fragment. Thank you for the explanation.